src/bin/cargo/commands/owner.rs

```diff
@@ -33,19 +76,24 @@ pub fn cli() -> Command {
 
 pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
     let registry = args.registry(config)?;
+
+    let Some((sc, arg)) = args.subcommand() else {
+    return Err(CliError::new(
+        anyhow::format_err!(
+            "you need to specify the subcommands to be operated: add, remove or list."
+        ),
+        101,
+    ));
+    };
```

Comment:
Breaking change and you can get clap to do stuff like this for you

Reply:
I have submitted an issue in Clap. Of course, I would also be happy to contribute to the clap if my suggestion was accepted.：)